### ISSUE TITLE
No run

### ISSUE THREAD

**Issue by artb1sh** [vyakhorev/SimpleBusiness#5]
raceback (most recent call last):
  File "main.py", line 78, in <module>
    run_crm_gui()
  File "main.py", line 31, in run_crm_gui
    from main_gui_modern import QtGui, gui_MainWindow
  File "/home/artb1sh/SimpleBusiness-master/main_gui_modern.py", line 13, in <module>
    from gui_forms_logic.dlgs.dlg_edit_contact import gui_DialogCrm_EditContact
  File "/home/artb1sh/SimpleBusiness-master/gui_forms_logic/dlgs/dlg_edit_contact.py", line 4, in <module>
    import db_main
  File "/home/artb1sh/SimpleBusiness-master/db_main.py", line 6, in <module>
    from db_crm import *
  File "/home/artb1sh/SimpleBusiness-master/db_crm.py", line 10, in <module>
    from cnf import user_name
  File "/home/artb1sh/SimpleBusiness-master/cnf.py", line 15, in <module>
    user_name = unicode(cnf.get("UserConfig", "UserName").decode("cp1251"))
  File "/usr/lib64/python2.7/ConfigParser.py", line 607, in get
    raise NoSectionError(section)
NoSectionError: No section: 'UserConfig'

**Comment by vyakhorev** [comment]
Hi there. 

Thank you for issue. This error is due to .ini absent. If you want to run it, make a directory named "__secret" in project directory and paste a file like this one there (rename to "main.ini"):

[main.txt](https://github.com/vyakhorev/SimpleBusiness/files/178639/main.txt)

Also you might have to create a directory named "temp" in the parent directory.

P.S.
This project is still under development. It is not yet a standalone application, it is integrated. So it is not simple to make it work. Notebook part works nice. If you have any questions, let me know. И можно по-русски.

**Comment by artb1sh** [comment]
Спасиб!

Думал подойдет прога для задумки, но что-то "списков"  нет. Из-за пустой бд? Или в бд другое приложение грузит?

![screenshot_20160318_012626](https://cloud.githubusercontent.com/assets/4118703/13863108/9a37f6ec-eca8-11e5-87cb-6f2c4c2a3133.png)

А в поиске первым было

**Comment by vyakhorev** [comment]
Да, списки из 1С гружу. Через xml. Не шустро, но подходит для маленького офиса. Скрипты чтения в проекте лежат. Если всё ещё интересно, могу формат xml дать.

Здесь ведем цены продажи и бюджет продажи (бизнес оптовой торговли). И контакты. Во вкладке "база знаний" ведем заметки - типа лиды/технические примечания  (там прикольный текстовый редактор, хештеги воспринимает). Но вообще в этой проге есть проблема - в ней пока блокировки не реализованы. Так что не знаю, как оно как макет пойдет..

А что за задумка?
